src/main/java/site/pages/LoginPage.java

```diff
@@ -0,0 +1,36 @@
+package site.pages;
+
+import com.codeborne.selenide.SelenideElement;
+import core.annotations.Name;
+import core.interfaces.PageElement;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+import org.openqa.selenium.support.FindBy;
+
+@Name("Login page")
```

Comment:
Why you use the same custom annotation for class and filed? This could lead to some misunderstanding in processing and logging

Reply:
This is Name annotation which is used to declare elements on the pages and pages by itself. Its pretty clear for me and I don't think it can lead to some misunderstandings just because this annotation represents Name of something. We just send some readable string to it so its good for visibility and logging.